### PR TITLE
Format rel path

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/25790

## Screenshots
```
/Users/bkatzir/dev/demisto/demisto-sdk/.tox/py37/bin/python /Users/bkatzir/dev/demisto/demisto-sdk/demisto_sdk/__main__.py format -i playbook-Akamai_WAF_-_Activate_Network_Lists.yml

======= Updating file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml =======
Removing sourceplaybookid field from playbook
Updating name of tasks who calls other playbooks to their name
Removing _dev and _copy suffixes from name and display tags
Removing Unnecessary fields from file
Setting JSON version to default: -1
Adding empty descriptions to relevant tasks
Saving output YML file to /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml 

Starting validating files structure
Could not find the old entity file under "/Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml".
please make sure that you did not break backward compatibility. Reason: Invalid URL '/Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml': No schema supplied. Perhaps you meant http:///Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml?
The files are valid
Format Status   on file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml - Success
Validate Status on file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml - Success

Process finished with exit code 0

```
## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
Screen Shot 2020-08-19 at 12 14 59

@jochman the test passes locally, what am I doing wrong with the `ChangeCWD` that it does not work in the build?

**Comment by jochman** [comment]
> Screen Shot 2020-08-19 at 12 14 59
> 
> @jochman the test passes locally, what am I doing wrong with the `ChangeCWD` that it does not work in the build?

Try using the rel_path instead of path.

**Comment by yaakovi** [comment]
looks like a bug:

> Could not find the old entity file under "/Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml".
please make sure that you did not break backward compatibility. Reason: Invalid URL '/Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml': No schema supplied. Perhaps you meant http:///Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* **format** now works with a relative path
```
Comment (yaakovi) [inline CHANGELOG.md:2]:
```suggestion
* **format** now works with a relative path.
```

**Comment by bakatzir** [comment]
@yaakovi You are right. I added the predefined_scheme using the `find_type` util.
```
======= Updating file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml =======
Removing sourceplaybookid field from playbook
Updating name of tasks who calls other playbooks to their name
Removing _dev and _copy suffixes from name and display tags
Removing Unnecessary fields from file
Setting JSON version to default: -1
Adding empty descriptions to relevant tasks
Saving output YML file to /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml 

Starting validating files structure
The files are valid
Format Status   on file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml - Success
Validate Status on file: /Users/bkatzir/dev/demisto/content/Packs/Akamai_WAF/Playbooks/playbook-Akamai_WAF_-_Activate_Network_Lists.yml - Success
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**